app/Controller/PagesController.php

```diff
@@ -40,7 +40,7 @@ class PagesController extends AppController {
 /**
  * Displays a view
  *
- * @return void
+ * @return \Cake\Network\Response|null
```

Comment:
You are right, but I think it's taken from [here](https://api.cakephp.org/2.9/source-class-Controller.html#757) :-)